xls/delay_model/op_module_generator.py

```diff
@@ -76,13 +78,32 @@ def generate_ir_package(op: str,
       ("bits[32]", "bits[16]").
     attributes: Attributes to include in the operation mnemonic. For example,
       "new_bit_count" in extend operations.
+    operand_attributes: Attributes consisting of an operand to include in the
```

Comment:
Add a test which uses this new functionality?

Reply:
See above (I used to have tests but removed them without removing all of the added functionality - now fixed).